src/transformers/models/realm/modeling_realm.py

```diff
@@ -875,8 +875,7 @@ def mask_to_score(mask):
         hidden_states = self.dense_intermediate(hidden_states)
         # [reader_beam_size, max_sequence_len, span_hidden_size]
         start_projection, end_projection = hidden_states.chunk(2, dim=-1)
-        block_mask = token_type_ids.detach().clone()
-        block_mask[:, -1] = 0
+
         candidate_starts, candidate_ends, candidate_mask = span_candidates(block_mask)
```

Comment:
We previously used `token_type_ids` as the `block_mask`, and now the `block_mask` is computed in the retriever.